dashboard/app/util_test.go

```diff
@@ -98,17 +98,25 @@ func (c *Ctx) expectFail(msg string, err error) {
 	}
 }
 
-func (c *Ctx) expectForbidden(err error) {
+func (c *Ctx) expectFailureStatus(err error, code int) {
 	c.t.Helper()
 	if err == nil {
-		c.t.Fatalf("expected to fail as 403, but it does not")
+		c.t.Fatalf("expected to fail as %d, but it does not", code)
 	}
 	httpErr, ok := err.(HTTPError)
-	if !ok || httpErr.Code != http.StatusForbidden {
-		c.t.Fatalf("expected to fail as 403, but it failed as %v", err)
+	if !ok || httpErr.Code != code {
+		c.t.Fatalf("expected to fail as %d, but it failed as %v", code, err)
 	}
 }
 
+func (c *Ctx) expectForbidden(err error) {
```

Comment:
Are you using expectForbidden anywhere?

Reply:
It is used in access_test.go, TestAccess() function.

Reply:
Ack.